packages/next/src/server/web/spec-extension/unstable-no-store.ts

```diff
@@ -30,10 +30,17 @@ export function unstable_noStore() {
     return
   } else {
     store.isUnstableNoStore = true
-    if (workUnitStore && workUnitStore.type === 'prerender') {
-      // unstable_noStore() is a noop in Dynamic I/O.
-    } else {
-      markCurrentScopeAsDynamic(store, workUnitStore, callingExpression)
+    if (workUnitStore) {
+      switch (workUnitStore.type) {
+        // TODO: make unstable_noStore only available in Server Components
```

Comment:
#75979 says:
> NOTE: For legacy/compat reasons, we're allowing unstable_cache and unstable_noStore, since they don't currently throw when called.

Reply:
yeah I suppose if it's already there we can just let it be